submissions/ayodeleb/HTML.TASK.html

```diff
@@ -0,0 +1,31 @@
+<!Doctype html>
+<html lang="en">
+    <head>
+        <title>perfectfit Login Page</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width initial-scale=1.2">
+    </head>
+    <body>
+        
+<h2>Welcome to perfectfit,this is an online shopping store for shoes, we provide comfort to  
+           your feet and make you look classy and chic.
+        </h4>
```

Comment:
Hello @joymeetme the opening tag and the closing tag should be the same <h2></h2> or <h4></h4>